examples/components/multi-select-view/multi-select-view.component.jsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+import { List } from 'immutable';
+import { MultiSelect } from '../../../src/index';
+
+export default class MultiSelectView extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { checkedItems: List() };
+  }
+
+  componentWillMount() {
+    this.items = this.initializeItems();
+  }
+
+  onChange = (id, isChecked) => {
+    let checkedItems = this.state.checkedItems;
```

Comment:
Using Map instead of a List with huge lists would be more efficient. Also we should agree that do we use Immutable.js with common components.